PyLeap_NeoPixel_demo/code.py

```diff
@@ -3,10 +3,27 @@
 import neopixel
 
 pixels = neopixel.NeoPixel(board.NEOPIXEL, 10, brightness=0.2, auto_write=False)
+rainbow_cycle_demo = 1
+
+def wheel(pos):
```

Comment:
We've taken to calling this `colorwheel`, as it is a clearer name. I'd really appreciate it if you could update the code to use `colorwheel` instead of `wheel`.
```suggestion
def colorwheel(pos):
```

Reply:
Good point! I'll update this.